misc-scripts/xref_mapping/XrefParser/Database.pm

```diff
@@ -57,11 +56,21 @@ sub verbose {
 }
 
 sub dbc {
-  my ($self, $arg) = @_;
-  (defined $arg) &&
-    ($self->{_dbc} = $arg );
+  my $self = shift;
+
+  if(@_) {
+    my $arg = shift;
+
+    if(defined($arg)) {
+      croak "$arg is not a DBConnection" 
+	unless $arg->isa('Bio::EnsEMBL::DBSQL::DBConnection');
```

Comment:
I thought this was an indent issue, but it's a tab. Please remove the tabs

Reply:
Done.